src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/AddTransparentDataEncryptionCertificateTests.ps1

```diff
@@ -0,0 +1,282 @@
+
+	try
+	{
+		Add-AzureRmSqlServerTransparentDataEncryptionCertificate `
+			-ResourceId $server.ResourceId `
+			-PrivateBlob $samplePrivateBlob
+	}
+	Catch{
+		Write-Debug "##### Following exception occurred causing test failure #####"
+		Write-Debug  $_.Exception.Message
+		#throw if we catch an exception
+		Assert-AreEqual $true $false
```

Comment:
Why not just use `throw` ?

Reply:
I added this just for ease of debugging, since I wasn't able to repro the test failure locally, once the travis build passes, will just remove the catch blocks.

Reply:
If it is useful for debugging now, it may be useful for debugging in the future. So you should not remove it. :)